code/modules/mob/mob_helpers.dm

```diff
@@ -30,6 +30,44 @@
 /mob/living/carbon/human/isMonkey()
 	return istype(species, /datum/species/monkey)
 
+
+/proc/ishuman_species(A)
+	if(istype(A, /mob/living/carbon/human) && (A:get_species() == "Human"))

```

Comment:
ishuman is already a proc that's fukken everywhere. Would be like pulling up a tree with all it's roots deep in the ground